### DISCUSSION TITLE
[v4] Cannot apply unknown utility class: !important

### DISCUSSION THREAD

**Discussion by tance77** [tailwindlabs/tailwindcss#15177] [Help] [answered]
What is the proper way to apply `!important` when using the `@apply` in tailwindcss v4?

In tailwindcss 3 I would simply do 

```
@apply bg-red-500 !important
```

Now in tailwindcss 4 I get an error saying

> Cannot apply unknown utility class: !important

**Comment by wongjn** [comment, marked as answer]
```css
@apply !bg-red-500;
```

**Reply by tance77** [reply]
That's freaking sweet!
